ui/src/pages/version/components/forms/components/CostEstimationPanel.js

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import {
+  EuiSpacer,
+  EuiCallOut,
+  EuiHorizontalRule,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+} from "@elastic/eui";
+import { calculateCost } from "../../../../../utils/costEstimation";
+
+/**
+ * Panel for showing cost estimation
+ * @param {VersionEndpoint} versionEndpoint version endpoint to be deployed
+ * @returns
+ */
+export const CostEstimationPanel = ({ versionEndpoint }) => {
+  const modelMinCost = calculateCost(
+    versionEndpoint.resource_request.min_replica,
+    versionEndpoint.resource_request.cpu_request,
+    versionEndpoint.resource_request.memory_request
+  );
+
+  const modelMaxCost = calculateCost(
+    versionEndpoint.resource_request.max_replica,
+    versionEndpoint.resource_request.cpu_request,
+    versionEndpoint.resource_request.memory_request
+  );
+
+  const transformerMinCost =
+    versionEndpoint.transformer && versionEndpoint.transformer.enabled
+      ? calculateCost(
+          versionEndpoint.transformer.resource_request.min_replica,
+          versionEndpoint.transformer.resource_request.cpu_request,
+          versionEndpoint.transformer.resource_request.memory_request
+        )
+      : 0;
+
+  const transformerMaxCost =
+    versionEndpoint.transformer && versionEndpoint.transformer.enabled
+      ? calculateCost(
+          versionEndpoint.transformer.resource_request.max_replica,
+          versionEndpoint.transformer.resource_request.cpu_request,
+          versionEndpoint.transformer.resource_request.memory_request
+        )
+      : 0;
+
+  const totalMinCost = modelMinCost + transformerMinCost;
+  const totalMaxCost = modelMaxCost + transformerMaxCost;
+
+  return (
+    <EuiCallOut title="Cost Estimation" iconType="tag">
+      <EuiSpacer size="m" />
+      <div>
+        <EuiFlexGroup>
+          <EuiFlexItem>
+            <h4>Model</h4>
+          </EuiFlexItem>
+          <EuiFlexItem style={{ textAlign: "right" }}>
+            {" "}
+            <h4>
+              USD {modelMinCost.toFixed(2)}- {modelMaxCost.toFixed(2)} / Month
```

Comment:
I'd rather use `$` symbol rather than USD and use lowercase month. It takes less space and more friendly to my eyes.

<img width="700" alt="image" src="https://user-images.githubusercontent.com/8122852/212872770-64475aed-0d22-4c0e-b7b9-c786bd94313f.png">

<img width="558" alt="image" src="https://user-images.githubusercontent.com/8122852/212873086-3964ce69-72d3-4aac-a8b6-b5c236ab257e.png">

Reply:
Anyway, I used this as reference: how Shopify recommends formatting currency: https://polaris.shopify.com/foundations/formatting-localized-currency